<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="ServerController">
    <property name="geometry">
     <rect>
      <x>291</x>
      <y>120</y>
      <width>211</width>
      <height>19</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue LED on/off</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="greenLed">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Green LED on/off</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>291</x>
      <y>101</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Servo </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>140</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>90</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>221</width>
      <height>63</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="StepperToggle">
       <property name="text">
        <string>Stepper motor </string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>100%</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Stepper Reverse</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QSlider" name="StepperSpeedController">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>9</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="UltSonToggle">
       <property name="text">
        <string>Ultrasonic Sensor </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="UtlSonMessure">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>LCD Display Message</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LCDlineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Send Text</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>534</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>pushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenLed</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>greenLedToggled()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>blueLedToggled()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ServerController</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>servoDegree(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>StepperToggle</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>stepperOnOff()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>stepperDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>StepperSpeedController</sender>
   <signal>actionTriggered(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>stepperSpeed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pushButtonClicked()</slot>
  <slot>greenLedToggled()</slot>
  <slot>blueLedToggled()</slot>
  <slot>servoDegree(int)</slot>
  <slot>stepperOnOff()</slot>
  <slot>stepperDir()</slot>
  <slot>stepperSpeed(int)</slot>
 </slots>
</ui>
